pytorch_lightning/metrics/classification.py

```diff
@@ -106,20 +102,20 @@ class ConfusionMatrix(TensorMetric):
     """
 
     def __init__(
-            self,
-            normalize: bool = False,
-            reduce_group: Any = None,
-            reduce_op: Any = None,
+        self,
+        normalize: bool = False,
+        reduce_group: Any = None,
     ):
         """
         Args:
             normalize: whether to compute a normalized confusion matrix
             reduce_group: the process group to reduce metric results from DDP
             reduce_op: the operation to perform for ddp reduction
```

Comment:
reduce_op got removed, doc string still has it
same in all other places where reduce_op got removed

Reply:
Will remove that tomorrow